package-res/reportviewer/paginationTemplate.html

```diff
@@ -238,6 +240,21 @@
     );
   }
 
+  function toLocaleStringSupportsLocales() {
+  var number = 0;
+  try {
+    number.toLocaleString('i');
+  } catch (e) {
+    return e.name === 'RangeError';
```

Comment:
The bug was in that that in PUC there is a comma, but in this template there no comma. As far as I understand in PUC comma added according to locale, so if we will add a comma in paginationTemplate.html in case there is no support toLocaleString() method than we will get different formats in PUC and paginationTemplate.html. And this will lead to a new bug. If I understand correctly all.